.circleci/config.yml

```diff
@@ -0,0 +1,66 @@
+version: 2.1
+
+workflows:
+  build-test:
+    jobs:
+      - prep-deps
+      - test-build:
+          requires:
+            - prep-deps
+      - test-lint:
+          requires:
+            - prep-deps
+      - all-tests-pass:
+          requires:
+            - test-build
+            - test-lint
```

Comment:
We should add a `test-unit` step here, following this format: https://github.com/MetaMask/eth-rpc-errors/blob/main/.circleci/config.yml#L64-L73

How you set up the script dependencies depend on what you do with the `test` script (see comment in `package.json`). If the `test` job includes the build step, there's no need for a separate `build` job.